UnityProject/Assets/Scripts/UI/SecurityRecords/GUI_SecurityRecordsEntryPage.cs

```diff
@@ -96,25 +96,25 @@ public void UpdateEntry()
 
 		if(characterSettings != null)
 		{
-			torso.SetComplicatedValue("human_parts_greyscale", characterSettings.torsoSpriteIndex, characterSettings.skinTone);
-			head.SetComplicatedValue("human_parts_greyscale", characterSettings.headSpriteIndex, characterSettings.skinTone);
+			torso.SetComplicatedValue("icons/mob/human_parts_greyscale", characterSettings.torsoSpriteIndex, characterSettings.skinTone);
+			head.SetComplicatedValue("icons/mob/human_parts_greyscale", characterSettings.headSpriteIndex, characterSettings.skinTone);
 			rightLeg.SetValue = characterSettings.skinTone;
 			leftLeg.SetValue = characterSettings.skinTone;
 			rightArm.SetValue = characterSettings.skinTone;
 			leftArm.SetValue = characterSettings.skinTone;
 			eyes.SetValue = characterSettings.eyeColor;
-			beard.SetComplicatedValue("human_face", characterSettings.facialHairOffset, characterSettings.facialHairColor);
-			hair.SetComplicatedValue("human_face", characterSettings.hairStyleOffset, characterSettings.hairColor);
+			beard.SetComplicatedValue("icons/mob/human_face", characterSettings.facialHairOffset, characterSettings.facialHairColor);
+			hair.SetComplicatedValue("icons/mob/human_face", characterSettings.hairStyleOffset, characterSettings.hairColor);
```

Comment:
Is this to make the loading a little bit better?
Since my only worry with this as it will make it a bit harder to move sprites around

Reply:
@Aranclanos can you clarify this?

Reply:
Resources.loadall requires a path to the subfolder within resources.
I don't have the time to check out the old folder, but (for example) just passing "human_face" to resources.loadall will not work...

For some reason Unity still hasn't made a fully recursive variant of resources.load* afaik :(

Reply:
I was using SpriteManager.PlayerSprites[string] but since SpriteManager doesnt contain all sprites I'm swapping to resource load, which needs the exact path

Reply:
Problem is, when we swap over to the new sprite system this will probably get overlooked and break. It would be best to add what you need to SpriteManager so there is only one place we need to fix instead of searching through code for these Resource references

Reply:
in that case this can be closed and left as it is